content/Access the world's most powerful retail APIs with Best Buy Ruby - Sandro Damilano.md

```diff
@@ -0,0 +1,43 @@
+## Access the world's most powerful retail APIs with Best Buy Ruby
+
+![Main image](images/blue_led_connection.jpg)
```

Comment:
Shouldn't we ask Sofi for help on the cover image?

Reply:
I have no idea haha do we?